.devcontainer/devcontainer.json

```
@@ -0,0 +1,26 @@
+{
+    "name": "openQA Single Instance",
+    "build": {
+        "dockerfile": "Dockerfile"
```

Can we use an image here? Since it takes a while to install everything from scratch. Something like this:

```suggestion
        "image": "registry.opensuse.org/devel/openqa/containers/openqa-single-instance"
```

---

Sure, that's one of the first things I tried.
Detailed answer in the ticket:
https://progress.opensuse.org/issues/87695?issue_count=54&issue_position=32&next_issue_id=58184&prev_issue_id=159408#note-18

---

What I also find a bit suboptimal with the single instance container is:
https://build.opensuse.org/projects/devel:openQA/packages/openQA-single-instance-container/files/_service:download_url:Dockerfile?expand=1
It installs openQA packages in the Dockerfile fron the tumbleweed repo:
```
RUN zypper in -y openQA-single-instance openQA-bootstrap \
    qemu-arm qemu-ppc qemu-x86 qemu-tools sudo iputils os-autoinst-distri-opensuse-deps
```
And when I later call openqa-bootstrap it adds the devel:openQA repo and installs this stuff again. Ok, it's quite fast because most dependencies are already there. But why was it done this way?